src/index.html

```diff
@@ -11,7 +11,176 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="group">
+        <legend>Personal information</legend>
+
+        <label class="form_field">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+            required
+          >
+        </label>
+
+        <label class="form_field">
+          Name:
+
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+            required
+          >
+        </label>
+
+        <label class="form_field">
+          How old are You?
+
+          <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+
+        <label class="form_field">
+          Full date of birth:
+
+          <input
+            type="date"
+            name="age"
+          >
+        </label>
+
+        <label class="form_field">
+          I accept the term of the agreement
+
+          <input
+            type="checkbox"
+            name="term"
+            required
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset class="group">
+        <legend>Registration</legend>
+        <label class="form_field">
+          E-mail:
+
+          <input
+            type="email"
+            name="mail"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+
+        <label class="form_field">
+          Password:
+
+          <input
+            type="password"
+            name="password"
+            minlength="5"
+            maxlength="32"
+            required
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset class="group">
+        <legend>An interesting fact about you!</legend>
+         Do you love cats?
+
+        <label>
+          <input
+            type="radio"
+            name="yes"
+          >
+          Yes
+        </label>
+        <label>
+          <input
+            type="radio"
+            name="no"
+          >
+          No
+        </label>
```

Comment:
wrap these guys to div and give it `class="form_field"`. It should help with the test. 
You didn't set any margin here, how it should work?